agent/eni/watcher/watcher_linux.go

```diff
@@ -57,6 +88,17 @@ type UdevWatcher struct {
 	primaryMAC           string
 }
 
+// unmanagedENIError is used to indicate that the agent found an ENI, but the agent isn't
```

Comment:
Can this be moved to another file?

Reply:
why?

Reply:
I was thinking we may need a `type.go` for the struct and const, to split this file into smaller pieces which are easier to read.

Reply:
i'm not a fan of the `type.go` model. It's much more easier to navigate/organize the code where the struct and the methods are defined in the same file. We could move this into an `errors.go` file. But, seeing how it's just one type, I'd rather keep it here than to create a new file for it.